web/client/components/widgets/widget/SingleWidgetControls.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+import Select from 'react-select';
+import { Glyphicon } from 'react-bootstrap';
+import classnames from 'classnames';
+
+import Button from "../../../components/misc/Button";
+
+const SingleWidgetControls = ({ options }) => {
+    const maxIndex = options.dropdownWidgets.length - 1;
+    const activeIndex = options.dropdownWidgets.findIndex(el => el.id === options.activeWidget.id);
+    const nextIndex = activeIndex === maxIndex ? 0 : activeIndex + 1;
+    const prevIndex = activeIndex === 0 ? maxIndex : activeIndex - 1;
+    return (
+        <div className="widget-selector">
+            {options.showArrowsNavigation ?
```

Comment:
please remove `options.showArrowsNavigation` is not requested in the issue

Reply:
removed